runtime/handler.go

```diff
@@ -92,26 +114,26 @@ func ForwardResponseStream(ctx context.Context, mux *ServeMux, marshaler Marshal
 		}
 		if err != nil {
 			grpclog.Infof("Failed to marshal response chunk: %v", err)
-			handleForwardResponseStreamError(ctx, wroteHeader, marshaler, w, req, mux, err)
+			handleForwardResponseStreamError(ctx, wroteHeader, marshaler, w, mux, isEventStream, err)
 			return
 		}
 
 		if isEventSource {
 			if writeEventHeader {
-				if _, err = fmt.Fprintf(w, "event: %s \n\n", eventHeader); err != nil {
+				if _, err = fmt.Fprintf(w, "event: %s \n", eventHeader); err != nil {
```

Comment:
According to [the spec](https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events) there is no empty line between the event type and the data.